index.html

```diff
@@ -80,14 +80,18 @@
         <div id="techtree">
             <div id="helptext">
                 <div id="helptext__content"></div>
+                <div id="helptext__advance_stats"></div>
                 <div id="helptext__x_ref"></div>
+                <div id="helptext__more">
```

Comment:
you could use `<details><summary>More info</summary> .... </details>` here instead and you'd get to drop most of the `toggle_more_stats` JS